applications/asset_tracker/src/main.c

```diff
@@ -1750,6 +1750,49 @@ void handle_bsdlib_init_ret(void)
 	#endif /* CONFIG_BSD_LIBRARY */
 }
 
+static void lte_handler(const struct lte_lc_evt *const evt)
+{
+	switch (evt->type) {
+	case LTE_LC_EVT_NW_REG_STATUS:
+		if ((evt->nw_reg_status != LTE_LC_NW_REG_REGISTERED_HOME) &&
+		     (evt->nw_reg_status != LTE_LC_NW_REG_REGISTERED_ROAMING)) {
+			break;
+		}
+
+		LOG_INF("Network registration status: %s",
```

Comment:
could do `"Network registration status: Connected - %s"`

Reply:
Will be reworked in a later PR including more event handling.